<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="repoLabel">
      <property name="font">
       <font>
        <family>Play</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Play&quot;;</string>
      </property>
      <property name="text">
       <string>Movie repository</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Play&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Short</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Detailed</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="font">
       <font>
        <family>Play</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="genreLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
          <property name="text">
           <string>Genre:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="yearLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="likesLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
          <property name="text">
           <string>Likes</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="trailerLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
          <property name="text">
           <string>Trailer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="titleEdit">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="genreEdit">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="yearEdit">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="likesEdit">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="trailerEdit">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Play&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Play&quot;;
</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="font">
         <font>
          <family>Play</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Play&quot;;</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Play&quot;;</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="undoButton">
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="redoButton">
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
